tests/format/js/for/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -683,47 +683,35 @@ label1: for (;;) continue label1
 ;
 
 =====================================output=====================================
-for (;;)
-  continue;
-  // comment
+for (;;) continue;
+// comment
 
-for (;;)
-  break;
-  // comment
+for (;;) break;
+// comment
```

Comment:
This is not a format change, but a pre-existing bug in the snapshot.
This aligns with the current main branch behavior.
https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAzCAnABACklAZxi1SwEsotgBfASiwCMM4BDAawG4AdKAel6yQAtkIQwQAGhAQADjDLQCyUCwwYIAdwAKqhEpQsANhpYBPJVKYswbODADKLUQBkKcZKiME4ljNdsOMtYUAObIMBgArj4gcEIMcAAmiUnOLFAhkSwhcABimEIsMPIZyCAskTAQkiAAFjBChgDqtWTwBEFgcPZ6bWQAbm2mZWAEFiAU3hgwWn4hhR5eMQBWBAAe9qGGcACKkRDwi4beUkEYU2UMLAmGNTIYFDBNZIkwtcgAHAAMp+reTX4ZGV7nApv13FIAI77eCzWT6coEAC0UDgSSSNWY0LIzFm2QWSE8xxi3iEZHCURJW12MPchKWUhg12er3eSAATIy-GRDKEAMIQEQsMqggCsNUi3gAKtd9ESTiB+tEAJJQFKwexgB5yACCavsMFM2yO3mo1CAA